README-CMAKE

```diff
@@ -153,6 +124,54 @@ We have two contributed CMake Build Environments:
 
   Exiv2 should be packaged in the dist directory with all the .lib, include and binary files you need.
 
+ 3 Running CMake commands manually from command line (new)
+
+  From Exiv2 0.26.0 to 0.26.1 the CMake code of the project was cleaned and now it is possible to configure
+  project from the command line as we do in the Unix sytems. As in the UNIX case, we can run the following
+  commands from the top directory to configure, build and install the library and utilities:
+
+    $ mkdir build && cd build
+    $ cmake .. or cmake-gui ..
+    $ make -j
```

Comment:
Remove the `make` commands from here. `make` is only available when we use the Makefile generator. For building code independently of the generator used we have to use:

```
cmake --build . --config Release
cmake --build . --target install
```